src/provisioner/provisioner.py

```diff
@@ -191,11 +200,11 @@ def _get_node_info(cortx_conf: MappedConf):
         return node_id, node_name
 
     @staticmethod
-    def _provision_components(cortx_conf: MappedConf, mini_prov_interfaces: list, apply_phase: str):
+    def _provision_components(cortx_conf: MappedConf, MINIPROVISIONING_STAGES: enum, apply_phase: str):
```

Comment:
There is no need to pass MINIPROVISIONING_STAGES. Its available as a constant.

Reply:
Even if it is available as constants, we need to pass it , as Upgrade also points to same function and there mini provisioning phase is only one, so provisioning_components has to accept mini provisioning stages based on deployment/upgrade.

Reply:
I think naming was the confusion. Pleaser rename this as 
```
class UpgradeInterfaces(Enum):
...

def _provision_components(cortx_conf: MappedConf, interfacces: Enum, apply_phase: str):
...
```

Reply:
done